curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md

```diff
@@ -18,11 +18,11 @@ localeTitle: Use Seletores de Atributo para Elementos de Estilo
 ```yml
 tests:
   - text: O seletor de atributo de <code>type</code> deve ser usado para selecionar as caixas de seleção.
-    testString: 'assert(code.match(/<style>[\s\S]*?\[type=("|")checkbox\1\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi),"The <code>type</code> attribute selector should be used to select the checkboxes.");'
+    testString: assert(code.match(/<style>[\s\S]*?\[type=("|")checkbox\1\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi));
```

Comment:
I suggest the text translated
```suggestion
    testString: 'assert(code.match(/<style>[\s\S]*?\[type=("|")checkbox\1\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi)," O seletor de atributo <code>type</code> deve ser usado para selecionar as caixas de seleção.");'
```

Reply:
We don't need to add assert messages.  Instead, the `text` section is the assert message on the preceding line should be adjusted as needed.